packages/@dcl/inspector/src/utils/tree.ts

```diff
@@ -0,0 +1,85 @@
+export type TreeType = 'directory' | 'file'
```

Comment:
There's no need to have different types of nodes in the tree to know if they can be expanded or collapsed, this can be known just by telling whether if a node is a leaf or not (ie. if it has children it can be expanded, otherwise not).